Documentation/common-problems.md

```diff
@@ -0,0 +1,128 @@
+WARNING    REQUEST_SLOW      1664 slow requests are blocked > 32 sec
+ERROR      REQUEST_STUCK     102722 stuck requests are blocked > 4096 sec
+WARNING    TOO_MANY_PGS      too many PGs per OSD (323 > max 300)
+WARNING    OSD_DOWN          1 osds down
+WARNING    OSD_HOST_DOWN     1 host (1 osds) down
+WARNING    PG_AVAILABILITY   Reduced data availability: 415 pgs stale
+WARNING    PG_DEGRADED       Degraded data redundancy: 190/958 objects degraded (19.833%), 53 pgs unclean, 53 pgs degraded, 53 pgs undersized
+
+USAGE:
+TOTAL        USED        DATA       AVAILABLE
+755.27 GiB   65.29 GiB   1.12 GiB   689.98 GiB
+
+MONITORS:
+NAME              ADDRESS               IN QUORUM   STATUS
+rook-ceph-mon75   172.18.0.70:6790/0    true        OK
+rook-ceph-mon21   172.18.0.245:6790/0   true        OK
+rook-ceph-mon65   172.18.0.246:6790/0   true        OK
+
+MGRs:
+NAME             STATUS
+rook-ceph-mgr0   Active
+
+OSDs:
+TOTAL     UP        IN        FULL      NEAR FULL
+4         1         2         false     false
+
+PLACEMENT GROUPS (600 total):
+STATE                        COUNT
+active+clean                 132
+active+undersized+degraded   53
+stale+active+clean           415
+```
+
+Another indication is when one or more of the MON pods restart frequently. Note the 'mon107' that has only been up for 16 minutes in the following output.
+
+```console
+$ kubectl -n rook get all -o wide --show-all
+NAME                                 READY     STATUS    RESTARTS   AGE       IP               NODE
+po/rook-api-41429188-x9l2r           1/1       Running   0          17h       192.168.1.187    k8-host-0401
+po/rook-ceph-mgr0-2487684371-gzlbq   1/1       Running   0          17h       192.168.224.46   k8-host-0402
+po/rook-ceph-mon107-p74rj            1/1       Running   0          16m       192.168.224.28   k8-host-0402
+rook-ceph-mon1-56fgm                 1/1       Running   0          2d        192.168.91.135   k8-host-0404
+rook-ceph-mon2-rlxcd                 1/1       Running   0          2d        192.168.123.33   k8-host-0403
+rook-ceph-osd-bg2vj                  1/1       Running   0          2d        192.168.91.177   k8-host-0404
+rook-ceph-osd-mwxdm                  1/1       Running   0          2d        192.168.123.31   k8-host-0403
+```
+
+### Solution
+What is happening here is that the MON pods are restarting and one or more of the CEPH daemons are not getting configured with the proper cluster information. This is commonly the result of not specifying a value for `dataDiskHostPath` in your Cluster CRD.
+
+The `dataDiskHostPath` setting specifies a path on the local host for the CEPH daemons to store configuration and data. Setting this to a path like `/var/lib/rook`, reapplying your Cluster CRD and restarting all the CEPH daemons (MON, MGR, OSD, RGW) should solve this problem. After the CEPH daemons have been restarted, it is advisable to restart the rook-api and rook-tool Pods. 
+
+## Reinstall fails to start all Pods
```

Comment:
What if this title is a little more specific such as this? `Only a single monitor pod starts`. I think that represents the main symptom.